docs/wiki/The library - Module usage.md

```diff
@@ -19,74 +15,164 @@ This section provides a guideline on how to use the CARML Bicep modules.
 
 This section shows you how to deploy a Bicep template.
 
-- [Deploy local template](#deploy-local-template)
-- [Deploy remote template](#deploy-remote-template)
+## PowerShell
 
-## Deploy local template
+This sub-section gives you an example on how to deploy a template from your local drive (file) or a publicly available remote location (URI).
 
-This sub-section gives you an example on how to deploy a template from your local drive.
-
-### **Local:** PowerShell
-
-This example targets a resource group level template.
+<details>
+<summary><i>Resource Group</i> scope</summary>
 
 ```PowerShell
 New-AzResourceGroup -Name 'ExampleGroup' -Location "Central US"
 
 $inputObject = @{
- DeploymentName    = 'ExampleDeployment'
- ResourceGroupName = 'ExampleGroup'
- TemplateFile      = "$home\ResourceModules\arm\Microsoft.KeyVault\vault\deploy.bicep"
+  DeploymentName    = 'ExampleDeployment'
+  ResourceGroupName = 'ExampleGroup'
+  # Using a local reference
+  TemplateFile      = "$home\ResourceModules\arm\Microsoft.KeyVault\vault\deploy.bicep"
```

Comment:
I think we're missing parameter files for this and all examples below

Reply:
Removed them on purpose as they are optional. However, will add them again if you think they proove useful. Linked to the official docs for all the extra parameters

Reply:
Added